tests/sdk/queries/fileevents/filters/test_event_filter.py

```diff
@@ -171,3 +171,45 @@ def test_source_not_in_str_gives_correct_json_representation():
     _filter = Source.not_in(items)
     expected = NOT_IN.format("source", *items)
     assert str(_filter) == expected
+
+
+def test_event_timestamp_choices_returns_valid_attributes():
+    choices = EventTimestamp.choices()
+    valid_set = {
+        "FIFTEEN_MINUTES",
+        "ONE_HOUR",
+        "THREE_HOURS",
+        "TWELVE_HOURS",
+        "ONE_DAY",
+        "THREE_DAYS",
+        "SEVEN_DAYS",
+        "FOURTEEN_DAYS",
+        "THIRTY_DAYS",
+    }
+    assert set(choices) == valid_set
+
+
+def test_event_type_choices_returns_valid_attributes():
+    choices = EventType.choices()
+    valid_set = {
+        "CREATED",
+        "MODIFIED",
```

Comment:
I thought this was `UPDATED` now? How is this test passing?

Reply:
The tests weren't validating the attributes and its values. Changed.